saw-central/src/SAWCentral/ImportAIG.hs

```diff
@@ -198,8 +192,9 @@ readAIG proxy opts sc f =
     let outLen = length outputLits
     inType <- scBitvector sc (fromIntegral inLen)
     outType <- scBitvector sc (fromIntegral outLen)
+    args <- mapM (\(x, t) -> scFreshEC sc x t) [("x", inType)]
```

Comment:
There is only one element in the list `[("x", inType)]`, so using `mapM` feels a bit silly. How about just:

```suggestion
    args <- scFreshEC sc "x" inType
```

This would require changing `args` to `[args]` below, but that's a pretty minor change.